CMakeLists.txt

```diff
@@ -487,7 +489,7 @@ if(WITH_CORE)
     set (HAVE_POSTGRESQL TRUE)
   endif()
 
-  set (WITH_QTWEBKIT TRUE CACHE BOOL "Enable QtWebkit support")
+  set (WITH_QTWEBKIT FALSE CACHE BOOL "Enable QtWebkit support")
```

Comment:
Shall we not make also QTWEBENGINE default to TRUE accordingly ?

Reply:
Done